src/CharacterAriaLive.js

```diff
@@ -15,53 +15,60 @@ type CharacterAriaLiveProps = {
 };
 
 class CharacterAriaLive extends React.Component<CharacterAriaLiveProps, {}> {
-    updateCharacterPositionAriaLive() {
-        const characterState = this.props.characterState;
-        const xPos = characterState.getColumnLabel();
-        const yPos = characterState.getRowLabel();
-        const direction = this.props.intl.formatMessage({id: `Direction.${characterState.direction}`});
-        const ariaLiveRegion = document.getElementById(this.props.ariaLiveRegionId);
+    getCharacterAriaLabel() {
         if (this.props.world === 'space') {
-            // $FlowFixMe: Flow doesn't know about character-position div
-            ariaLiveRegion.innerText=this.props.intl.formatMessage(
-                {id:'ProgramBlockEditor.spaceShipCharacter'},
-                {
-                    xPos,
-                    yPos,
-                    direction
-                }
+            return this.props.intl.formatMessage(
+                {id:'ProgramBlockEditor.spaceShipCharacter'}
             );
         } else if (this.props.world === 'forest') {
-            // $FlowFixMe: Flow doesn't know about character-position div
-            ariaLiveRegion.innerText=this.props.intl.formatMessage(
-                {id:'ProgramBlockEditor.rabbitCharacter'},
-                {
-                    xPos,
-                    yPos,
-                    direction
-                }
+            return this.props.intl.formatMessage(
+                {id:'ProgramBlockEditor.rabbitCharacter'}
             );
         } else {
-            // $FlowFixMe: Flow doesn't know about character-position div
-            ariaLiveRegion.innerText=this.props.intl.formatMessage(
-                {id:'ProgramBlockEditor.robotCharacter'},
-                {
-                    xPos,
-                    yPos,
-                    direction
-                }
+            return this.props.intl.formatMessage(
+                {id:'ProgramBlockEditor.robotCharacter'}
             );
         }
     }
 
+    clearCharacterPositionAriaLive() {
```

Comment:
This method isn't clearing the character position, but rather setting the text to "{character} is moving". Please rename this method to better match its behaviour.

Reply:
I tried to avoid using words like "announce", I went with `setCharacterMovingAriaLive`.